force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.js

```diff
@@ -0,0 +1,91 @@
+import { LightningElement, track } from 'lwc';
+const devFundWeight = 0.23;
+const processAutoWeight = 0.3;
+const userInterfaceWeight = 0.25;
+const testDebugWeight = 0.22;
+
+export default class PlatformDevCertCalculator extends LightningElement {
+
+    devFundamentals = 50;
+    processAutomation = 50;
+    userInterface = 50;
+    testDebugDeploy = 50;
+
+    certificationScore = 90;
+    numberOfQuestionsCorrect = 60;
+
+    showResources = false;
+
+    currentHistoryId = 0;
+
+    @track attemptHistory = [
+        {Id: 1, Score:50},
+        {Id: 2, Score:25}
+    ];
+
+    calculateScore(){
+        let devFundWeightScore = this.devFundamentals * devFundWeight;
+        let processAutoWeightScore = this.processAutomation * processAutoWeight;
+        let userInterfaceWeightScore = this.userInterface * userInterfaceWeight;
+        let testDebugWeightScore = this.testDebugDeploy * testDebugWeight;
+
+        this.certificationScore = devFundWeightScore + processAutoWeightScore + userInterfaceWeightScore + testDebugWeightScore;
+        console.log(this.certificationScore);
+
+        this.showResourcesIfFailed();
+        this.addAttemptHistory(this.certificationScore);
+    }
+
+    handleChange(event){
```

Comment:
You can add a validation on handle change so it add a toast error message when the use puts in invalid inputs like -100 or 123123123